<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiAcqCfgDialogClass</class>
 <widget class="QDialog" name="MultiAcqCfgDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiAcqCfgDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1,1,1,0">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="ElaText" name="label_2">
     <property name="text">
      <string>采集张数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ElaSpinBox" name="spinBox_n">
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="ElaCheckBox" name="checkBox_saveToFiles">
     <property name="text">
      <string>保存到文件</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="ElaText" name="label">
     <property name="text">
      <string>保存位置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="ElaLineEdit" name="lineEdit_savePath"/>
   </item>
   <item row="2" column="4">
    <widget class="ElaPushButton" name="pushButton_select">
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="ElaPushButton" name="pushButton_confirm">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="ElaPushButton" name="pushButton_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ElaSpinBox</class>
   <extends>QSpinBox</extends>
   <header>ElaSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ElaPushButton</class>
   <extends>QPushButton</extends>
   <header>ElaPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>ElaText</class>
   <extends>QLabel</extends>
   <header>ElaText.h</header>
  </customwidget>
  <customwidget>
   <class>ElaCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ElaCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ElaLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ElaLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
